docs/services/invocation.md

```diff
@@ -141,38 +141,60 @@ The Invocation identifier is opaque and its current format should not be relied
 
 ## Cancel an invocation
 
-:::caution
+If an invocation takes too long to complete and, therefore, is no longer of interest, you can cancel it.
```

Comment:
```suggestion
If an invocation takes too long to complete or is no longer of interest, you can cancel it.
```
There can be many reasons for cancelling besides taking too long, I guess.

Reply:
True. Will change it.